src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Et3ResponseHelper.java

```diff
@@ -129,90 +140,128 @@ private static void setTitle(Et3ResponseData data, String title) {
             return;
         }
 
-        switch  (title) {
-            case "Mr":
-                data.setTitleMr(CHECKED);
-                break;
-            case "Mrs":
-                data.setTitleMrs(CHECKED);
-                break;
-            case "Miss":
-                data.setTitleMiss(CHECKED);
-                break;
-            case "Ms":
-                data.setTitleMs(CHECKED);
-                break;
-            default:
+        switch (title) {
+            case "Mr" -> data.setTitleMr(CHECKED);
```

Comment:
I know you did it because sonarLint suggested that but then PMD was not happy with this so you suppressed PMD violation. It's a catch 22 so we can leave the switch case as it is.